include/ipc/ipc_service.h

```diff
@@ -68,6 +68,9 @@ struct ipc_ept_cfg {
 
 	/** Private user data. */
 	void *priv;
+
+	/** Name of the target instance. */
+	const char *target_inst;
```

Comment:
Adding a new API that looks things up by string names feels a bit weird in "modern" Zephyr, by which I mean post-v2.5 Zephyr. We have everything we need to go from devicetree nodes to real devices, and this sure "smells" like a new device API, but I'm new to this area and so this is just intuition talking.